### ISSUE TITLE
@context and @type are not in Information Model

### ISSUE THREAD

**Issue by egekorkan** [w3c/wot-thing-description#394]
Following presentation of @mkovatsc at Princeton Testfest:

`@context` and `@type` are important keywords but they are presented only in the serialization section. Information Model should be where they are introduced.

**Comment by takuki** [comment]
We still need to discuss which additional classes we have (and do not have) use cases to annotate with @type.

**Comment by mmccool** [comment]
@takuki made some commits to add `@context` and `@type` but he only edited template.txt, so they are not actually in the formal information model. In the Feb 15 meeting we decided they should be (and not treated as exceptions, but like any other element, including their handling of default values, etc). @vcharpenay volunteered to update the spec so that these are in the TTL files and so incorporated into the information model (including, for example, the automatically generated diagrams).

**Comment by mmccool** [comment]
I should mention I also edited @takuki contribution to make the ids unique for his new assertions to align with testing needs.  However, I did feel that for @context an extra assertion needs to be added around the phrase about optional prefixes... and that this statement should also be broken out into a separate "MAY" assertion.   Also I think (or rather, agree with others that brought this up in the meeting) extra assertions like this should not be nested inside the table, but instead should be stated separately in the text.  In fact rows of tables are treated as assertions in the implementation report.  They probably should focus on defining vocabulary (otherwise we get nested assertions in the implementation report...)
